src/components/MidiMessageTable.tsx

```
@@ -110,5 +118,69 @@ const MessageData = ({ messageEvent, formattedMessage }: MessageDataParams) => {
     </Group>
   }
 
-  return <Text>{messageEvent.data.map(b => b.toString(16).toUpperCase()).join(", ")}</Text>
+  return <Text>{_.map(messageEvent.data, formatHex).map((d) => <Code mx={2}>{d}</Code>)}</Text>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)